previous.txt

```diff
@@ -0,0 +1 @@
+547436 hash
```

Comment:
Is this equivalent to selecting the oldest game entry in the `games` table that doesn't have ratings yet? Wouldn't that be preferable over using an external file to tell us where to start the rating algorithm?

Reply:
Currently the Tak server always saves a completed game with a default adjusted rating then the rating script comes through and updates to what the correct adjusted rating is which is difficult to ensure we could use that to allow us to see what games have or haven't been rated. 

In the future we can make a change to the games db to start tracking if a game has actually been rated or not which would allow us to remove the need for saving the last game ID.